integration_tests/test_get_requests.py

```diff
@@ -43,6 +42,14 @@ def check_response(r: Response):
     check_response(get(f"/{function_type}/template"))
 
 
+@pytest.mark.benchmark
+@pytest.mark.parametrize("function_type", ["sync", "async"])
+def test_url_for(function_type: str):
+    r = get(f"/{function_type}/url_for")
+    assert r.status_code == 200
+    assert r.text == f"/{function_type}/url_for"
```

Comment:
this should not pass. is this passing for you?

And this is not the correct assertion. https://github.com/sparckles/Robyn/pull/758/files#diff-1276ef09a4b09c2f539dfc4e497e5054f03f2388440a91da0354e74f7fa828cdR71-R76

You need to check if this altered text is present in the rendered html.